### DISCUSSION TITLE
[iOS Seminar] Swift Package Manager를 통한 SnapKit import

### DISCUSSION THREAD

**Discussion by gsanpark** [wafflestudio/seminar-2023#144] [Q&A] [answered]
// 참고 자료: https://weekoding.tistory.com/9

Seminar 1에서 AutoLayout을 더 편하게 활용할 수 있는 패키지로 SnapKit을 소개해주셨는데,
세미나에서 시연해주셨던 것처럼 File > Add Package > Search로 검색하더라도 SnapKit 패키지가 검색되지 않습니다.

<img width="1031" alt="SPM" src="https://github.com/wafflestudio/seminar-2023/assets/37293866/436302c4-9c75-4dbe-be30-8a073b688314">

이럴 때는 어떻게 조치할 수 있을까요?

**Comment by gsanpark** [comment, marked as answer]
SnapKit과 같이 Apple Swift Packages에 포함되어 있지 않은 패키지는 github repo 주소를 기입하는 형식으로 추가할 수 있는 것으로 보입니다.

SnapKit github repo: https://github.com/SnapKit/SnapKit.git

세미나장님께서는 과거에 SnapKit을 사용하신 적이 있어 SPM에서 곧바로 검색되었던 것으로 보입니다.

<img width="1024" alt="image" src="https://github.com/wafflestudio/seminar-2023/assets/37293866/c27eeb24-8ea1-4b77-8163-3ec552271b83">

P.S.) Labels를 어떻게 설정할 수 있는지 모르겠습니다.. 죄송합니다..🥲

**Reply by shp7724** [reply]
첫 검색 시에는 반드시 `*.git` 형태의 url을 사용해야 하는군요. 질문&답변 감사합니다! 🙇‍♂️
